ftw/upgrade/command/__init__.py

```diff
@@ -142,7 +144,25 @@ def __call__(self):
         args = self.parser.parse_args()
         configure_logging(args)
         setattr(args, 'parser', self.parser)
-        args.func(args)
+        if getattr(args, 'all_sites', False):
+            info = {}
+            while True:
+                try:
+                    with capture() as out:
+                        args.func(args)
+                    output = out.getvalue().strip()
+                    if getattr(args, 'json', False):
+                        # "[]" must be turned into [].
+                        output = json.loads(output)
+                    info[args.picked_site] = output
+                except StopIteration:
+                    break
+            if getattr(args, 'json', False):
+                print(json.dumps(info))
```

Comment:
When using `-L --json`, the JSON is "pretty", but with `-A --json` it is compact.
Can we use the pretty arguments here too so that we have consistent output? :wink: 
[`json.dumps(info, indent=4, encoding='utf-8')`](https://github.com/4teamwork/ftw.upgrade/blob/f6c6847f47afdfab71ced57f8db6da6dbadb32b9/ftw/upgrade/jsonapi/utils.py#L116)